src/components/AddressText/index.tsx

```diff
@@ -0,0 +1,63 @@
+import { Tooltip } from 'antd'
+import { FC } from 'react'
+import classNames from 'classnames'
+import { Link, LinkProps } from 'react-router-dom'
+import { useBoolean } from '../../utils/hook'
+import EllipsisMiddle from '../EllipsisMiddle'
+import CopyTooltipText from '../Text/CopyTooltipText'
+import styles from './styles.module.scss'
+
+const AddressText: FC<{
+  children: string
+  className?: string
+  containerClass?: string
+  disableTooltip?: boolean
+  linkProps?: LinkProps
+  monospace?: boolean
+  useTextWidthForPlaceholderWidth?: boolean
+}> = ({
+  children: address,
+  className,
+  containerClass,
+  disableTooltip,
+  linkProps,
+  monospace = true,
+  useTextWidthForPlaceholderWidth = true,
+}) => {
+  const [truncated, truncatedCtl] = useBoolean(false)
```

Comment:
use `isTruncated` to imply it's a `boolean`

Reply:
pushed